test/unit/model/config-test.js

```diff
@@ -0,0 +1,28 @@
+import Model from 'controller/model';
+
+describe('Model', function() {
+    const config = {
+        file: '//playertest.longtailvideo.com/adaptive/bbbfull/bbbfull.m3u8',
+        image: 'http://d3el35u4qe4frz.cloudfront.net/bkaovAYt-480.jpg',
+        preload: 'auto'
+    };
+
+    let model;
+
+    before(function () {
+        model = new Model();
+        model.setup(config);
+    });
+
+    describe('return value', function() {
+        it.only('should return an object on setup', function() {
+            expect(model).to.be.an('object');
```

Comment:
Since you're instantiating `model`, it would be an object. A better test would be to confirm that the configs passed in were added to the model.